Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+    ln_s buildpath, buildpath/"src/github.com/bitrise-io/bitrise"
+
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "_bin/bitrise"
+
+    bin.install "_bin/bitrise"
+  end
+
+  test do
+    output = shell_output(bin/"bitrise --version")
```

Comment:
`stepman` is not required but in a normal use case it'll be used, that's why it's part of the `bitrise setup` process to get it.

`envman` could also be optional, it's required for the current `bitrise` version but technically it's also optional; but again, in a normal setup the user will want to have it available, because `envman` is responsible for exposing environment variables between independent steps, so a subsequent step can access a previous step's exposed environments.